.pre-commit-config.yaml

```diff
@@ -1,18 +1,21 @@
+default_language_version:
+  python: python3.8  # Set this to the earliest supported Python version
+
 repos:
 
   - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+    rev: 5.13.2
```

Comment:
I really wish we could tie this to the stuff we have in a requirements.txt file somewhere 😐

Reply:
TBH I'm not a fan of pre-commit myself...

Reply:
I really don't like the way that you configure pre-commit, but pre-commit.ci is brilliant since it just fixes stuff instead of hassling people to run tooling locally, and worth putting up with a lot of these hassles.  Well I guess you're putting up with them on our behalf right now but *hypothetically* it would be worth me putting up with them as well :)

Reply:
The thing is that `black` is not dependency of Twisted.

pre-commit is our only tool for code reformating.

I think that for now, we should just leave with this.

In the future, we can look at replacing this with `ruff`